### PR TITLE
add a panel and make it responsive

### PR DESCRIPTION
**Pull request by pricealexandra**
it's not pretty but it's something!

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -2,7 +2,9 @@ import Head from "next/head";
 import { Map } from "@/components/map";
 import { Header } from "@/components/header";
 import theme, { SourceSans } from "@/utils/theme";
-import { useFetchBikeShare } from '@/hooks/useFetchBikeShare';
+import { useFetchBikeShare } from "@/hooks/useFetchBikeShare";
```
Comment (pricealexandra) [inline pages/index.tsx:5]: ugh I think your ide is doing single quote formatting and mine is double...we should probably add this to the list of things for the pre-commit hook to decide

Comment (paigewilliams) [reply pages/index.tsx:5]: oh no sorry!

**Comment by pricealexandra** [comment]
@paigewilliams are you thinking it will make sense for me to merge this PR, or are you replacing it with the mantine work you are doing?

**Comment by paigewilliams** [comment]
> @paigewilliams are you thinking it will make sense for me to merge this PR, or are you replacing it with the mantine work you are doing?

I think that we can merge this. Just from looking at what the Mantine AppShell offers out of the box, it doesn't seem like we can have the panel contents move easily to a bottom div on mobile. So I think that we should merge this and I can see what we can do with the AppShell in a branch?

**Review by paigewilliams — APPROVED** [review]

**Comment by pricealexandra** [comment]
> > @paigewilliams are you thinking it will make sense for me to merge this PR, or are you replacing it with the mantine work you are doing?
> 
> I think that we can merge this. Just from looking at what the Mantine AppShell offers out of the box, it doesn't seem like we can have the panel contents move easily to a bottom div on mobile. So I think that we should merge this and I can see what we can do with the AppShell in a branch?

ok, that sounds good to me! maybe app shell can be a later experiment if we run into issues styling this version?

**Pull request merged**